.isort.cfg

```diff
@@ -1,6 +1,6 @@
 [settings]
 line_length=79
-skip=nameko/cli/run.py,nameko/click_cli/run.py
+skip=nameko/cli/do_run.py
```

Comment:
Why rename this module from `run` to `do_run`?

Reply:
The intention was to keep modules related to actions of subcommand clearly different. I reverted the change so it is named `run.py` and `shell.py` as before.